frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -495,6 +497,15 @@ export const dashboardLogic = kea<dashboardLogicType>([
                         : state,
             },
         ],
+        maybeUrlVariables: [
```

Comment:
Does this become no longer `maybe` because we're parsing the variables from the URL from `parseURLVariables`?